### PR TITLE
 Generic assembly

### PR DESCRIPTION
**Pull request by yunfangjuan**

### CODE REVIEW

**Comment by yunfangjuan** [comment]
a few more tweaks but can be reviewed.

**Comment by tfrcarvalho** [comment]
Can you provide a description?

**Comment by yunfangjuan** [comment]
New workflow:
Please 
1. ignore reclassify_reads.py
2. see the sample new work flow execution at examples/assemble_blast.json and see the new classes that are created and how they are executed.

File: idseq_dag/steps/run_assembly.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import os
+import traceback
+from collections import defaultdict
+from idseq_dag.engine.pipeline_step import PipelineStep
+import idseq_dag.util.command as command
+import idseq_dag.util.log as log
+import idseq_dag.util.count as count
+
+class PipelineStepRunAssembly(PipelineStep):
+    ''' PriceSeq PipelineStep implementation '''
```
Comment (cdebourcy) [inline idseq_dag/steps/run_assembly.py]: priceseq?

File: examples/assemble_blast.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
+{
+  "output_dir_s3": "s3://idseq-samples-prod/yunfang/sample_1793/assembly_blast/",
```
[the remark this answers is not in the document]
Comment (jameshowardwang) [reply examples/assemble_blast.json]: yes, let's please put them in idseq-samples-development or somewhere else! let's keep that idseq-samples-prod bucket pristine

**Pull request merged**